### ISSUE TITLE
.field private transient elements:[Ljava/lang/Object; panic: runtime error: index out of range

### ISSUE THREAD

**Issue by Uriziel01** [AlexeySoshin/smali2java#6]
Hi,
I have this problem when trying to disassemble attached `smali.zip`:
```
.field private transient elements:[Ljava/lang/Object;
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/alexeysoshin/smali2java/parser.(*FieldParser).Parse(0xc0003dd868, 0xc0003dd930, 0xc0003f3740, 0x4, 0x4, 0xc24, 0x0)
        C:/go_work/src/github.com/alexeysoshin/smali2java/parser/field.go:63 +0x9e8
github.com/alexeysoshin/smali2java/parser.(*JavaFile).ParseLine(0xc0003dd930, 0xc000386e40, 0x35, 0xc24, 0xc000386e40)
        C:/go_work/src/github.com/alexeysoshin/smali2java/parser/java_file.go:107 +0x8d5
main.convertSmali(0xc00039a180, 0x5c, 0xc0000560b0)
        C:/go_work/src/github.com/AlexeySoshin/smali2java/smali2java.go:64 +0x1f2
main.parseSmaliFiles.func1(0xc00039a180, 0x5c, 0x509fa0, 0xc000208850, 0x0, 0x0, 0xc0003ddaf8, 0x485b3f)
        C:/go_work/src/github.com/AlexeySoshin/smali2java/smali2java.go:36 +0xb8
path/filepath.walk(0xc00039a180, 0x5c, 0x509fa0, 0xc000208850, 0xc0000422d0, 0x0, 0x0)
        C:/Go/src/path/filepath/path.go:358 +0x410
path/filepath.walk(0xc0001db0e0, 0x4b, 0x509fa0, 0xc000337dc0, 0xc0000422d0, 0x0, 0x0)
        C:/Go/src/path/filepath/path.go:382 +0x2f2
path/filepath.walk(0xc0001daf50, 0x42, 0x509fa0, 0xc000337ce0, 0xc0000422d0, 0x0, 0x0)
        C:/Go/src/path/filepath/path.go:382 +0x2f2
path/filepath.walk(0xc0003d49c0, 0x3a, 0x509fa0, 0xc0003141c0, 0xc0000422d0, 0x0, 0x0)
        C:/Go/src/path/filepath/path.go:382 +0x2f2
path/filepath.walk(0xc00008600f, 0x36, 0x509fa0, 0xc000092000, 0xc0000422d0, 0x0, 0x10)
        C:/Go/src/path/filepath/path.go:382 +0x2f2
path/filepath.Walk(0xc00008600f, 0x36, 0xc0000422d0, 0x1, 0x0)
        C:/Go/src/path/filepath/path.go:404 +0x10c
main.parseSmaliFiles(0xc00008600f, 0x36)
        C:/go_work/src/github.com/AlexeySoshin/smali2java/smali2java.go:29 +0x90
main.main()
        C:/go_work/src/github.com/AlexeySoshin/smali2java/smali2java.go:23 +0x94
```
[smali.zip](https://github.com/AlexeySoshin/smali2java/files/2797988/smali.zip)

**Comment by John-K** [comment]
I'm also seeing a similar error. Let me know if an additional test case would be helpful

**Comment by AlexeySoshin** [comment]
Should be resolved by #16
